ci-operator/step-registry/image-based-install-operator/post/gather/image-based-install-operator-post-gather-commands.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+echo "************ image based install operator post gather command ************"
+
+source "${SHARED_DIR}/packet-conf.sh"
+
+ssh "${SSHOPTS[@]}" "root@${IP}" bash - << "EOF"
+
+# prepending each printed line with a timestamp
+exec > >(awk '{ print strftime("[%Y-%m-%d %H:%M:%S]"), $0 }') 2>&1
+
+set -xeo pipefail
+
+cd /root/dev-scripts
+source common.sh
+source utils.sh
+source network.sh
+
+export ARTIFACT_DIR=/ibio-post-gather
+mkdir -p ${ARTIFACT_DIR}
+oc get baremetalhost ostest-extraworker-0 -n openshift-machine-api -o yaml > ${ARTIFACT_DIR}/baremetalhost.yaml
```

Comment:
Try to get the dataimage as well (if it fails just ignore it), if it wasn't deleted we would like to see the CR